source/launch.txt

```diff
@@ -24,13 +24,16 @@ databases. You have several options for creating a schema and
 launching :binary:`~bin.mongosqld`. This guide will help you choose
 the best option for your needs.
 
-By default, :binary:`~bin.mongosqld` generates a :doc:`data schema
-</schema-configuration>` and holds it in memory. Alternatively, if
-you prefer to create a schema file and edit it manually, use the
-:binary:`~bin.mongodrdl` program to create a ``.drdl`` schema
-file and use the :option:`--schema <mongosqld --schema>` option
-when starting :binary:`~bin.mongosqld`. For more information on schema
-generation and data sampling, see :doc:`schema-configuration`.
+By default, :binary:`~bin.mongosqld` generates a default :doc:`data schema
+</schema-configuration>` and stores it in memory. To create a custom schema
+instead and edit it manually, perform both of the following actions:
+
+- Use the :binary:`~bin.mongodrdl` program to create a custom ``.drdl`` schema
```

Comment:
Replace the new line 31 and 32 with the existing text, i.e.:
use the [mongodrdl](https://www.mongodb.com/docs/bi-connector/master/reference/mongodrdl/#mongodb-binary-bin.mongodrdl) program to create a .drdl schema file and use the [--schema](https://www.mongodb.com/docs/bi-connector/master/reference/mongosqld/#std-option-mongosqld.--schema) option when starting [mongosqld](https://www.mongodb.com/docs/bi-connector/master/reference/mongosqld/#mongodb-binary-bin.mongosqld).